app/routes/snippets-management/edit/edit-snippet.js

```diff
@@ -12,4 +12,9 @@ export default class SnippetsManagementEditSnippetRoute extends Route {
     });
     return container[0];
   }
+
+  setupController(controller, model) {
+    super.setupController(controller, model);
+    controller._editorDocument = null;
+  }
```

Comment:
I'm guessing this isn't needed anymore either? :)

Reply:
Yes, you are totally right, good catch